modules/storages/lib/api/v3/file_links/create_endpoint.rb

```diff
@@ -0,0 +1,38 @@
+module API::V3::FileLinks
+  class CreateEndpoint < API::Utilities::Endpoints::Create
+    include ::API::V3::Utilities::Endpoints::V3Deductions
+    include ::API::V3::Utilities::Endpoints::V3PresentSingle
+
+    def process(request, params_elements)
+      global_result = ServiceResult.new(
+        success: true,
+        result: []
+      )
+      params_elements.each do |params|
+        one_result = super(request, params)
+        global_result.add_dependent!(one_result)
+        global_result.result << one_result.result
+      end
+      global_result
+    end
+
+    def present_success(request, call)
```

Comment:
`call` is an awful name, as the base class of `call` is `ServiceResult`. Hard to make any connection between both without introspecting code. I copy-pasted original code from `API::Utilities::Endpoints::Bodied#present_success` without really thinking about choosing better names.

It would be easier if ruby was typed :)

Reply:
I tend to go with `service_call` nowadays which works for me. I also tried out `result` but that read awfully when calling `result.result`

Reply:
I'll rename as `service_call` 👍️